File: Arrays101/1051. Height Checker.java
class Solution {
    public int heightChecker(int[] heights) {
        if(heights.length<2) return 0;
        int[] heights2 = heights.clone();
        Arrays.sort(heights2);
        int result = 0;
        for(int x=0; x<heights.length; x++){
            if(heights[x]!=heights2[x]) result++;
        }
        return result;
    }
}

File: Arrays101/1051. Height Checker.py
class Solution:
    def heightChecker(self, heights: List[int]) -> int:
        if len(heights) < 2: return 0
        list2 = sorted(heights)
        lcomp = [i for i in range(len(heights)) if heights[i] != list2[i]]
        """
        result = 0
        for (x,y) in zip(heights, list2):
            if(x!=y): result+=1
        """
        return len(lcomp)
        